packages/jest-expect-openapi/package.json

```diff
@@ -4,8 +4,17 @@
   "description": "Jest/Vitest matcher for asserting valid OpenAPI definitions",
   "license": "MIT",
   "author": "Jon Ursenbach <jon@readme.io>",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "sideEffects": false,
```

Comment:
Will bump this to a major release if you also think https://github.com/readmeio/oas/pull/958#discussion_r1982408949 should be too.

Reply:
confirmed `node10` still works so im not going to treat this as a breaking release